_search-plugins/search-relevance/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Search relevance
+nav_order: 55
+has_children: false
+has_toc: false
+redirect_from:
+  - /search-plugins/search-relevance/
+---
+
+# Compare search results
+
+Compare search results is an experimental feature. For updates on the progress of Compare search results and other search relevance features, or if you want to leave feedback that could help improve the feature, join the [discussion on the OpenSearch forum](https://forum.opensearch.org/t/feedback-experimental-feature-compare-search-results/11331).    
+{: .warning}
+
+Compare search results is the first search relevance feature in OpenSearch. It lets you compare search results from two queries side by side to determine if one query produces better results than the other. Using this tool, you can evaluate search quality through experimenting with queries. 
+
+For example, you can see how results change when you apply one of the following query changes:
+
+- Weighting different fields differently.
+- Different stemming or lemmatization strategies.
+- Shingling.
+
+## Prerequisites
+
+Before you get started, you must index data in OpenSearch. To learn how to create a new index, see [Index data]({{site.url}}{{site.baseurl}}/opensearch/index-data). 
+
+Alternatively, you can add sample data in OpenSearch Dashboards using the following steps:
+
+1. On the top menu bar, go to **OpenSearch Dashboards > Overview**.
+1. Select **View app directory**.
+1. Select **Add sample data**.  
+1. Choose one of the built-in datasets and select **Add data**.
+
+
+## Using search relevance in OpenSearch Dashboards
+
+To compare search results in OpenSearch Dashboards, perform the following steps.
+
+**Step 1:** On the top menu bar, go to **OpenSearch Plugins > Search Relevance**.  
+
+**Step 2:** Enter the search text in the search bar.
+
+**Step 3:** Select an index for **Query 1** and enter a query (request body only) in [OpenSearch Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl). The `GET` HTTP method and the `_search` endpoint are implicit. Use the `%SearchText%` variable to refer to the text in the search bar.
+
+The following is an example query:
+
+```json
+{
+  "query": {
+    "multi_match": {
+      "query": "%SearchText%",
+      "fields": [ "title", "text" ]
+    }
+  }
+}
+```
+
```

Comment:
Should this line space (line 58) be before step 4? For example, step 5 doesn't have a line space before it.